src/tools/annotation/RectangleRoiTool.js

```diff
@@ -351,12 +351,16 @@ function _calculateRectangleStats(sp, rectangle) {
   let sumSquared = 0;
   let count = 0;
   let index = 0;
+  let min = sp ? sp[0] : null;
+  let max = sp ? sp[0] : null;
 
   for (let y = rectangle.top; y < rectangle.top + rectangle.height; y++) {
     for (let x = rectangle.left; x < rectangle.left + rectangle.width; x++) {
       sum += sp[index];
       sumSquared += sp[index] * sp[index];
-      count++;
+      min = Math.min(min, sp[index]);
+      max = Math.max(max, sp[index]);
+      count++; // TODO: Wouldn't this just be sp.length?
```

Comment:
Considering the fact that `sum += sp[index]` isn't checking to see if `index` is out of bounds, and `count` and `index` should always be the same, I think you're right.  I think `count` will always end up being the same as `sp.length` after the `for` loops.  I also don't think `count` is even used once it's returned from both `_calculateRectangleStats` and  `_calculateStats`.